test/traffic_cop/traffic_cop_test.cpp

```diff
@@ -133,6 +133,7 @@ TEST_F(TrafficCopTests, BasicTest) {
                                             port_, catalog::DEFAULT_DATABASE));
 
     pqxx::work txn1(connection);
+    txn1.exec("DROP TABLE IF EXISTS TableA");
```

Comment:
Is this still needed?

Reply:
No, this is not needed. I'd like to remove this.